### DISCUSSION TITLE
vite2+vue3+antdv3+ts项目报Failed to resolve component: AInput

### DISCUSSION THREAD

**Discussion by rainmanhhh** [vueComponent/ant-design-vue#5542] [Q&A] [answered]
如果不import，直接用AInput, AButton这种自带的别名，就报Failed to resolve component: AInput
如果手动import {Input} from 'ant-design-vue'，则idea不识别组件props，在template里面无法自动补全

**Comment by geeklibin** [comment, marked as answer]
其实你可以使用插件`unplugin-vue-components`，无需在任何地方引入组件，直接在template里使用就好了<a-input>
